src/Polysemy/Final/Error.hs

```diff
@@ -13,28 +13,28 @@ import           Polysemy.Final
 import           Polysemy.Error
 
 ------------------------------------------------------------------------------
--- | Run an 'Error' effect as an 'IO' 'Exception'.
+-- | Run an 'Error' effect as an 'IO' 'Exception' through 'Final' 'IO'.
```

Comment:
When this PR moves into polysemy proper, is the plan to merge eg `Polysemy.Final.Error` into `Polysemy.Error`?

Reply:
That's my thought, yeah.